PrintMessage.cpp

```diff
@@ -0,0 +1,53 @@
+#include "./PrintMessage.h"
+#include <thread>
+#include <chrono>
+#include <iostream>
+#include <map>
+
+using std::cout, std::flush, std::this_thread::sleep_for, std::chrono::seconds;
+
+Language currentLanguage = ENGLISH;
+
+std::map<Language, std::map<std::string, std::string>> messages = {
```

Comment:
You have an enum for Language, so typos in the Language will be caught by the compiler itself. 
Why not make the message id (like `HYPO_THERMIA`) also enum instead of strings?